Source/3rdParty/duktape/duktape.h

```diff
@@ -176,16 +182,16 @@
  * development snapshots have 99 for patch level (e.g. 0.10.99 would be a
  * development version after 0.10.0 but before the next official release).
  */
-#define DUK_VERSION                       20600L
+#define DUK_VERSION                       20700L
 
 /* Git commit, describe, and branch for Duktape build.  Useful for
  * non-official snapshot builds so that application code can easily log
  * which Duktape snapshot was used.  Not available in the ECMAScript
  * environment.
  */
-#define DUK_GIT_COMMIT                    "fffa346eff06a8764b02c31d4336f63a773a95c3"
-#define DUK_GIT_DESCRIBE                  "v2.6.0"
-#define DUK_GIT_BRANCH                    "v2-maintenance"
+#define DUK_GIT_COMMIT                    "03d4d728f8365021de6955c649e6dcd05dcca99f"
+#define DUK_GIT_DESCRIBE                  "03d4d72-dirty"
```

Comment:
seems off with the git repository is dirty

Reply:
@project64  That's weird because it is just the 2.7.0 release copied over